src/commands/codepush/lib/release-command-skeleton.ts

```diff
@@ -145,6 +147,14 @@ export default class CodePushReleaseCommandSkeleton extends AppCommand {
     }
   }
 
+  private printWarningForPoorTargetBinaryVersionIfNeeded(version: string): void {
+    const warningVersion = generateWarningVersionForPoorVersionIfNeeded(version);
+
+    if (warningVersion) {
+      out.text(`\nYour target-binary-version "${version}" will be treated as "${warningVersion}".\n`);
```

Comment:
My thoughts that I'm not sure that it can be important for users if this is done on the server side or with cli and possible this info can be superfluous. But it isn't a big issue and we can add this info. =)

Reply:
No worries, it wasn't a big thing - just thinking about whether this will confuse users :) Let's leave it as it is for now. If we find any users getting in touch to seek clarification we can always update it later.

Reply:
Sure, absolutely :)
Thanks!